src/Http/StringStream.php

```diff
@@ -0,0 +1,75 @@
+<?php
+namespace LORIS\Http;
+
+/**
+ * A StringStream provides a simple wrapper over a string to convert
+ * it to a \Psr\Http\Message\StreamInterface
+ */
+class StringStream implements \Psr\Http\Message\StreamInterface {
+    // @todo Handle error cases correct. PSR7 says to throw an exception,
+    // but the fopen/etc functions generally return the false literal.
+    protected $stream;
+    protected $size;
+
+    public function __construct($val) {
+        $this->stream = fopen("php://temp", "r+");
+        fwrite($this->stream, $val);
+        rewind($this->stream);
+        $this->size = strlen($val);
+    }
+    public function __toString() {
```

Comment:
> This method MUST NOT raise an exception in order to conform with PHP's string casting operations.
according to the specs

Reply:
Diactoros implementation of Stream put the function code in a try catch and return an empty string when an exception is catched.

Reply:
I don't think it was possible for it to throw an exception before, but it's simple enough to do that, so I added it.